application/src/test/java/org/thingsboard/server/controller/EdgeControllerTest.java

```diff
@@ -901,9 +903,13 @@ public void testSyncEdge() throws Exception {
         edgeImitator.ignoreType(OAuth2ClientUpdateMsg.class);
         edgeImitator.ignoreType(OAuth2DomainUpdateMsg.class);
 
-        edgeImitator.expectMessageAmount(27);
+        // 18 connect message
```

Comment:
maybe we should avoid ignoring UserCredentialsUpdateMsg?
this was initially, because UserCredentialsUpdateMsg was sent additionally, but not UserCredentialsUpdateMsg is send with user update message.
I recommend to remove line 
edgeImitator.ignoreType(UserCredentialsUpdateMsg.class);
and check test behaviour.